Sources/CleverBird/chat/streaming/StreamOptions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  StreamOptions.swift
+//
+//
+//  Created by Ronald Mannak on 5/6/24.
```

Comment:
Feel free to leave this Created by line though. :)

Reply:
@btfranklin Done!